### PR TITLE
Improve json section

### PR DESCRIPTION
**Pull request by mplewis**
I've added the following to the JSON section:

* How to use simplejson in place of json in older versions of Python
* How to load and save JSON from/to file objects instead of strings

I've also reworded the intro paragraph a little and cleaned up the language of the simplejson paragraph.

Please take a look and tell me what you think! Thanks.

### CODE REVIEW

File: docs/scenarios/json.rst [hunk 1]
```diff
@@ -39,10 +39,48 @@ You can also convert a the following to JSON:
     print(json.dumps(d))
     '{"first_name": "Guido", "last_name": "Rossum", "titles": ["BDFL", "Developer"]}'
 
+Working with File Objects
+-------------------------
```
Comment (sigmavirus24) [inline docs/scenarios/json.rst]: Didn't we discuss in an earlier PR that we weren't going to cover working with file objects?

Comment (mplewis) [inline docs/scenarios/json.rst]: I didn't see any such discussion in the previous JSON PRs.

Why not cover working with file objects? Parsing JSON from a file and writing JSON to an output file is a pretty common use case of the JSON library.

Comment (sigmavirus24) [inline docs/scenarios/json.rst]: Please read the review comments on https://github.com/kennethreitz/python-guide/pull/511

Comment (mplewis) [inline docs/scenarios/json.rst]: Got it, thanks! I couldn't see the comments on the outdated diffs before.
I'll remove the file object sections.
ᐧ

On Tue, Feb 10, 2015 at 10:00 PM, Ian Cordasco <notifications@github.com>
wrote:

> In docs/scenarios/json.rst
> <https://github.com/kennethreitz/python-guide/pull/515#discussion_r24473205>
> :
>
> > @@ -39,10 +39,48 @@ You can also convert a the following to JSON:
> >      print(json.dumps(d))
> >      '{"first_name": "Guido", "last_name": "Rossum", "titles": ["BDFL", "Developer"]}'
> >
> > +Working with File Objects
> > +-------------------------
>
> Please read the review comments on #511
> <https://github.com/kennethreitz/python-guide/pull/511>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kennethreitz/python-guide/pull/515/files#r24473205>.
>



-- 
Matthew Lewis
mplewis.com | (612) 600-6907

**Comment by mplewis** [comment]
Just removed the section on file objects—let me know if you find anything else you think should be changed.

**Comment by sigmavirus24** [comment]
Thanks @mplewis !

**Pull request merged**